pandas/core/generic.py

```diff
@@ -1194,44 +1199,54 @@ def rename_axis(self, mapper, axis=0, copy=True, inplace=False):
 
     def _set_axis_name(self, name, axis=0, inplace=False):
         """
-        Alter the name or names of the axis.
+        Alter the label(s) of the axis.
```

Comment:
This doesn't seem correct. Altering the labels would be changing the names of the columns, not the name of the axis itself.

And small thing, but alter doesn't necessarily need to be correct. If the axis doesn't have a name, it's setting it, but not altering. `Set` would be better I think.